packages/safe-tools-client/app/components/network-connect-modal/index.gts

```diff
@@ -0,0 +1,197 @@
+          }}
+
+          as |Section ActionChin|
+        >
+          <button
+            class='network-connect-modal__close-button'
+            type='button'
+            aria-label='Close'
+            {{on 'click' @onClose}}
+          >
+            {{svgJar 'close' width='100%' height='100%' aria-hidden=true}}
+          </button>
+
+          <Section @title="Connect your wallet">
+            <BoxelRadioInput
+              @name="wallet-provider"
+              @orientation="horizontal"
+              @spacing="default"
+              @groupDescription='Select a wallet to connect to'
+              @items={{this.walletProviders}}
+              @disabled={{@isConnecting}}
+              @checkedId={{@currentWalletProviderId}}
+              @hideRadio={{true}}
+              class='network-connect-modal__wallet-group'
+              data-test-wallet-selection as |option|
+            >
+              {{#let option.data as |item|}}
+                <option.component
+                  @name='wallet-provider-selection'
+                  @onChange={{fn (optional @changeWalletProvider) item.id}}
+                  @disabled={{not item.enabled}}
+                  data-test-wallet-option={{item.id}}
+                >
+                  <div class='network-connect-modal__wallet-item'>
+                    <img
+                      src={{item.logo}}
+                      alt=''
+                      role='presentation'
+                      class='network-connect-modal__wallet-item-image'
+                    />
+                    <span
+                      class='network-connect-modal__wallet-item-name'
+                    >
+                      {{item.name}}
+                    </span>
+                    {{#if item.explanation}}
+                      <span
+                        class='network-connect-modal__wallet-item-description'
+                      >
+                        {{item.explanation}}
+                      </span>
+                    {{/if}}
+                  </div>
+                </option.component>
+              {{/let}}
+            </BoxelRadioInput>
+          </Section>
+
+          <ActionChin @state='default'>
+            <:default as |a|>
+              <a.ActionButton {{on "click" this.connect}} data-test-mainnet-connect-button>
+                Connect Wallet
+              </a.ActionButton>
+            </:default>
+            <:inProgress as |i|>
+              <i.ActionStatusArea class="network-connect-modal__in-progress-logo" @icon={{concat
+                this.radioWalletProviderId "-logo" }} style={{cssVar status-icon-size="2.5rem" }}>
+                <BoxelLoadingIndicator class="network-connect-modal__loading-indicator" @color="var(--boxel-light)" />
+                <div class="network-connect-modal__waiting-status">
+                  Waiting for you to connect your {{!-- network-display-info "layer1" "conversationalName" --}} wallet...
```

Comment:
Layer 1 doesn't seem to be a user-facing concept in this app. Should we remove the commented text?

Reply:
ya sure hehe done